public/js/tmapp.js

```diff
@@ -80,6 +82,51 @@ const tmapp = (function() {
         _updateURLParams();
     }
 
+    function setBrightness(b) {
+        _currState.brightness = b;
+        _updateBrightnessContrast();
+    }
+
+    function setContrast(c) {
+        _currState.contrast = c;
+        _updateBrightnessContrast();
+    }
+
+    /**
+     * Apply _currState brightness/contrast to current viewer
+     */
+    function _updateBrightnessContrast() {
+
+        function contrast_brightness(c,b) {
+            return function(x) {return Math.pow(10,c)*(x-128)+128 + b*128;}
```

Comment:
Small detail, but maybe this should be changed in the future to accommodate for images with different color depths.